spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/EventProcessorProperties.java

```diff
@@ -165,6 +177,41 @@ public void setInitialSegmentCount(int initialSegmentCount) {
             this.initialSegmentCount = initialSegmentCount;
         }
 
+        /**
+         * See {@link TrackingEventProcessorConfiguration#andTokenClaimInterval(long, java.util.concurrent.TimeUnit)}
+         * @return the tokenClaimInterval
+         */
+        public long getTokenClaimInterval() {
+            return tokenClaimInterval;
+        }
+
+
```

Comment:
Nit: I don't think you need this empty line here.

Reply:
I do agree. Done.